### DISCUSSION TITLE
[Feature]

### DISCUSSION THREAD

**Discussion by leo719044** [YetiForceCompany/YetiForceCRM#15057] [How to]
🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅

Can I enable the attachment upload in the comment section under ACCOUNT / OPPORTUNITIES/ TICKETS??

**Comment by rskrzypczak** [comment]
Currently, Yetiforce does not have a functionality that allows you to upload files to comments.
